src/core/Tooltip/Tooltip.text.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
Just a note to remember for us to rename the file to be `Tooltip.test.tsx`, small typo in name as it is not `.text.`.

Reply:
Renamed.